IdentityCore/tests/automation/ui_tests_lib/MSIDTestConfigurationProvider.m

```diff
@@ -174,11 +182,30 @@ - (MSIDAutomationTestRequest *)defaultAppRequest:(NSString *)environment
     
     NSString *testEnvironment = environment ? environment : self.wwEnvironment;
     
-    request.requestScopes = [self scopesForEnvironment:testEnvironment type:@"ms_graph"];
-    request.requestResource = [self resourceForEnvironment:testEnvironment type:@"ms_graph"];
-    request.expectedResultScopes = [NSString msidCombinedScopes:request.requestScopes withScopes:[self scopesForEnvironment:testEnvironment type:@"oidc"]];
+    if (self.testsConfig.scopesSupported)
```

Comment:
Can you provide some context on why this transformation is needed?

Reply:
It is needed for MSAL CPP integration, they don't support scopes.